test/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ macro (add_app_test test_filename test_alternative)
 
     # Create the test target.
     add_executable (${target} ${test_filename})
-    target_link_libraries (${target} "${PROJECT_NAME}_lib" seqan3::seqan3 gtest gtest_main)
+    target_link_libraries (${target} "${PROJECT_NAME}_lib" seqan3::seqan3 gtest gtest_main seqan_app_coverage)
```

Comment:
Is this needed? "${PROJECT_NAME}_lib" already has `seqan_app_coverage` set. So it will pull that recursively in

Reply:
Yes :)
This was because I fixed the tests first :D